_episodes/04-lists.md

```diff
@@ -129,31 +129,31 @@ does not.
 > the list value, it will change for both variables!
 >
 > ~~~
-> salsa = ['peppers', 'onions', 'cilantro', 'tomatoes']
-> my_salsa = salsa        # <-- my_salsa and salsa point to the *same* list data in memory
-> salsa[0] = 'hot peppers'
-> print('Ingredients in my salsa:', my_salsa)
+> mlld_salsa = ['peppers', 'onions', 'cilantro', 'tomatoes']
+> hot_salsa = mild_salsa        # <-- mild_salsa and hot_salsa point to the *same* list data in memory
+> hot_salsa[0] = 'hot peppers'
+> print('Ingredients in mild salsa:', mild_salsa)
```

Comment:
Could you please make sure this and a few other lines below conform to the proposed change of (salsa ~> mild_salsa) and (my_salsa ~> hot_salsa)?

Reply:
I went back through and did an updated patch that I believe caught any remaining references later in the episode.

Reply:
Thanks, @ndporter, but my point was that you proposed to change `salsa` to `mild_salsa` and `my_salsa` to `hot_salsa`, whereas this line changes `my_salsa` to `mild_salsa`.